tests/cephfs/cephfs_vol_management/cephfs_vol_mgmt_fs_create_delete_loop.py

```diff
@@ -0,0 +1,231 @@
+                raise CommandFailed("Failed to create nfs cluster")
+            log.info("ceph nfs cluster created successfully")
+
+            nfs_export_name = "/export_" + "".join(
+                secrets.choice(string.digits) for i in range(3)
+            )
+
+            subvol_path_nfs, rc = client1.exec_command(
+                sudo=True,
+                cmd=f"ceph fs subvolume getpath {fs_name} {subvolume_name}_3 {subvolume_group_name}",
+            )
+            export_path = f"{subvol_path_nfs}"
+            nfs_mounting_dir = "/mnt/nfs_" + "".join(
+                secrets.choice(string.ascii_uppercase + string.digits) for i in range(5)
+            )
+
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export create cephfs {nfs_name} "
+                f"{nfs_export_name} {fs_name} path={export_path}",
+            )
+
+            rc = fs_util.cephfs_nfs_mount(
+                client1, nfs_server, nfs_export_name, nfs_mounting_dir
+            )
+            if not rc:
+                log.error("cephfs nfs export mount failed")
+                return 1
+            mounting_dir = "".join(
+                random.choice(string.ascii_lowercase + string.digits)
+                for _ in list(range(10))
+            )
+            client1.exec_command(sudo=True, cmd=f"mkdir -p {nfs_mounting_dir}/nfs_dir")
+            client1.exec_command(
+                sudo=True,
+                cmd=f"python3 /home/cephuser/smallfile/smallfile_cli.py --operation create "
+                f"--threads {threads} --file-size {file_size} --files {files} --response-times Y --top "
+                f"{nfs_mounting_dir}/nfs_dir",
+                long_running=True,
+            )
+
+            for subvolume in subvolume_list:
+                subvolume_size_subvol = fs_util.get_subvolume_info(
+                    client=client1, **subvolume
+                )
+                log.info(f"Subvolume Info after IO: {subvolume_size_subvol}")
+
+        except Exception as e:
+            log.info(e)
+            log.info(traceback.format_exc())
+            return 1
+
+        finally:
+            fs_util.client_clean_up(
+                "umount", fuse_clients=[client1], mounting_dir=fuse_mount_dir
+            )
+
+            fs_util.client_clean_up(
+                "umount",
+                kernel_clients=[client1],
+                mounting_dir=kernel_mount_dir,
+            )
+
+            client1.exec_command(sudo=True, cmd=f"rm -rf {nfs_mounting_dir}/*")
+            client1.exec_command(sudo=True, cmd=f"umount {nfs_mounting_dir}")
+            client1.exec_command(
+                sudo=True, cmd=f"rm -rf {nfs_mounting_dir}/", check_ec=False
+            )
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export delete {nfs_name} {nfs_export_name}",
+                check_ec=False,
+            )
+
+            fs_util.remove_fs(client1, fs_name)
```

Comment:
We are deleting FS without deleting subvolumes and subvolumegroups. Is this expected for this workflow?

Reply:
Done